src/components/scenes/existingAccout/ChangeRecoveryScene.tsx

```diff
@@ -78,7 +78,7 @@ export const ChangeRecoveryScene = (props: Props) => {
   const stringPredicate = (s: string | undefined | null): s is string =>
     typeof s === 'string'
 
-  const handleQuestion = (index: number) => () => {
+  const handleQuestion = (index: number) => async () => {
```

Comment:
You need to `await` the `Airship.show` call below, and remove the `.then(...)` wrapper around the rest of the function.